### PR TITLE
add-last-sync-time-unknown

### PR DESCRIPTION
**Pull request by udaraweerasinghege**
Fixes this: https://github.com/mozilla/fxa-content-server/issues/4988

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: d73ce5ef527288f4b3c3737e3941465795204e55
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 3fefc6dcbf08a72c3f08c7a1254541b2d40d9cfc
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

File: app/scripts/views/settings/clients.js [hunk 1]
```diff
@@ -76,7 +76,7 @@ define(function (require, exports, module) {
           }
         } else {
           // unknown lastAccessTimeFormatted or not possible to format.
-          item.lastAccessTimeFormatted = '';
+          item.lastAccessTimeFormatted = t('last sync time unknown');
```
Comment (shane-tomlinson) [inline app/scripts/views/settings/clients.js]: Hmm, is there no unit test for this? If not, can you create one?

File: app/scripts/views/settings/clients.js [hunk 2]
```diff
@@ -76,7 +76,7 @@ define(function (require, exports, module) {
           }
         } else {
           // unknown lastAccessTimeFormatted or not possible to format.
-          item.lastAccessTimeFormatted = '';
```
Comment (vladikoff) [inline app/scripts/views/settings/clients.js]: let's just leave this `''`

File: app/scripts/views/settings/clients.js [hunk 3]
```diff
@@ -76,7 +76,7 @@ define(function (require, exports, module) {
           }
         } else {
```
Comment (vladikoff) [inline app/scripts/views/settings/clients.js]: Will chat with @ryanfeeley about it, we need to tweak the `t('started syncing ` string instead

**Comment by vladikoff** [comment]
Let's revisit this later

**Pull request closed without merging**